smaug/python/ops/activation_ops_test.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, *args, **kwargs):
     self.tensor_shape = [2, 32, 32, 32]
 
   def do_basic_test(self, test_graph, node_name, op_type, tensor_shape=None):
-    node = test_graph.get_node(node_name)
+    graph_proto, tensor_data_array = test_graph.to_proto()
```

Comment:
I don't believe `tensor_data_array` is used here, so replace with `_`.

Reply:
Done.